ssl-impl/src/main/java/io/crate/protocols/ssl/SslConfiguration.java

```diff
@@ -84,7 +85,7 @@ static SslContext buildSslContext(Settings settings) {
                 trustManagers = trustStoreSettings.get().trustManagers;
             }
 
-            SSLContext sslContext = SSLContext.getInstance("TLS");
+            SSLContext sslContext = SSLContext.getInstance("TLSv1.2");
```

Comment:
Would it make sense to extract the changes here to a separate PR against master? Seem to be independent of HTTPS